Source/control.cpp

```diff
@@ -1691,11 +1692,13 @@ void ReleaseChrBtns(bool addAllStatPoints)
 	}
 }
 
-static int DrawDurIcon4Item(CelOutputBuffer out, ItemStruct *pItem, int x, int c)
+#define DURABILITY_RED 2

```

Comment:
I feel like `DURABILITY_RED/GOLD` are very poor names for these, as they represent the outcome and not the actual semantic.

Can we come up with better names to represent these values? Perhaps something like `WeakenedItemDurabilityThreshold` and `NearBreakingItemDurabilityThreshold`?

Reply:
Pretty sure the C way is to call it `gcbDurRd`